src/FeedGenerator.php

```diff
@@ -629,39 +711,35 @@ public function handle_action_timeout( $action_id, $error ) {
 	 * @since 1.2.14
 	 */
 	protected function is_timeout_error( array $error ): bool {
-		return isset( $error['type'] ) &&
-				E_ERROR === $error['type'] &&
-				isset( $error['message'] ) &&
-				strpos( $error ['message'], 'Maximum execution time' ) !== false;
+		return isset( $error['type'] ) && E_ERROR === $error['type'] &&
+			isset( $error['message'] ) && strpos( $error ['message'], 'Maximum execution time' ) !== false;
```

Comment:
To improve readability, I think it would make sense to rework this method a bit. Here's my idea:

```suggestion
		$is_error              = E_ERROR === $error['type'] ?? 0;
		$is_max_execution_time = strpos( $error['message'] ?? '', 'Maximum execution time' ) !== false;

		return $is_error && $is_max_execution_time;
```